_locales/en/messages.json

```diff
@@ -242,8 +242,13 @@
 		"message": "Enables websites to publish, distribute, and optimize audio and video content."
 	},
 	"category_cdn": {
+		"description": "Ghostery organizes tags by category. This is one of several tag categories.",
 		"message": "CDN"
 	},
+	"category_cdn_desc": {
+		"description": "User-facing description for the Social Media tag category.",
```

Comment:
Most new categories have the "User-facing description for the Social Media tag category" description. Is this intended?

If a generic one is sufficient, perhaps "Ghostery organizes tags by category. This is one of several tag categories" (as in the others)?

Reply:
this is a translation key for a description, not for the category name

Reply:
But still, does it make sense to use "Social Media tag category" everywhere for the new ones? I don't see the link to the CDN or telemetry category?

For the others that we had before, it made more sense, for example:

```
"category_advertising_desc": {
		"description": "User-facing description for the Advertising tag category.",`
```

Reply:
oh, now I see.